Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM java:openjdk-7-jdk
+
+RUN apt-get update
+ENV CLOUD_SDK_VERSION 161.0.0
+RUN apt-get -qqy update && apt-get install -qqy \
+        curl \
+        gcc \
+        python-dev \
+        python-setuptools \
+        apt-transport-https \
+        lsb-release \
+        openssh-client \
+        git \
+    && easy_install -U pip && \
+    pip install -U crcmod   && \
```

Comment:
I added a comment to the top of the file explaining that this section is copy/pasted from Google's dockerfile. (We can't just use their image out of the box because we *also* need to use the Java7 dockerfile; we have to pick one to build on, and graft the other in manually.)